armbian/base/build/customize-armbian-rockpro64.sh

```diff
@@ -200,6 +203,10 @@ EOF
 # retain journal logs between reboots 
 ln -sf /mnt/ssd/system/journal/ /var/log/journal
 
+# SYSTEM CONFIGURATION ---------------------------------------------------------
+SYSCONFIG_PATH="/opt/shift/sysconfig/"
+mkdir -p $SYSCONFIG_PATH
+echo "BITCOIN_NETWORK=testnet" > ${SYSCONFIG_PATH}BITCOIN_NETWORK
```

Comment:
I would call the file we write to `bitcoin_network.env` or `bitcoin_network.conf` or similar, to a) give a hint through the file suffix that the file is a text file for environment variables / configuration and b) use lower case.

(And elsewhere.)

Reply:
I could live with the `.env` suffix, although that makes it not as clean to write and source the values in my opinion.

Why would you prefer the lowercase filename if the containing variable is uppercase?

Reply:
> I could live with the .env suffix, although that makes it not as clean to write and source the values in my opinion.

I'm not sure I understand what makes it less clean to do e.g `source bitcoin_network.env`, instead of `source bitcoin_network`.

> Why would you prefer the lowercase filename if the containing variable is uppercase?

UNIX style file naming usually uses lower-case filenames. I don't believe there's any reason why the file name's case has to match the contents of that file; e.g `.bashrc` is lower-case, even if it might set/modify environment variables with upper-case names, like `PATH` for example.

Reply:
> Why would you prefer the lowercase filename if the containing variable is uppercase?

Ah, now I see what you were going for — you define a single variable in each file, instead of a set of variables related to a certain service or piece of infrastructure (e.g Bitcoin, or networking).

Sure, if you prefer to have `$FOO` defined in the file `FOO` and to keep a 1:1 mapping between file containing environment variables and the environment variables defined within, I won't object. I'm not sure it's a very common way to manage configuration, but as long as we stay consistent and it works well for you, go ahead.